src/status_im2/common/mute_chat_drawer/view.cljs

```diff
@@ -0,0 +1,67 @@
+(ns status-im2.common.mute-chat-drawer.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo2]
+            [react-native.core :as rn]
+            [status-im2.constants :as constants]
+            [utils.re-frame :as rf]
+            [status-im2.common.mute-chat-drawer.style :as style]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+(defn mute-chat-drawer [chat-id accessibility-label]
+  (fn []
+    [rn/view {:accessibility-label accessibility-label}
+     [quo2/text {:weight :medium
```

Comment:
You can just alias as `quo`, `quo2` is redundant inside `status_im2`, because eventually we'll remove the old `quo` and the name `quo2` won't make sense anymore.